### PR TITLE
Bug 1153351 - Move Firefox for Android release notes to /firefox/android/

### PR DESCRIPTION
**Pull request by kyoshino**
Need this once https://github.com/mozilla/bedrock/pull/2906 is merged.

### CODE REVIEW

**Comment by jgmize** [comment]
@kyoshino thanks for writing these tests, I believe they have revealed an issue with https://github.com/mozilla/bedrock/pull/2906 -- likely we will need to fix something in global.conf -- see https://people.mozilla.org/~jmize/bedrock/mobile-to-android-release-notes.html#sec-2-2 for more details.

File: tests/test_redirect.py [hunk 1]
```diff
@@ -62,10 +66,31 @@ def test_redirect_community_to_contribute(self, mozwebqa):
...
+                                '/mobile/beta/notes/',
+                                '/firefox/android/beta/notes/',
+                                False)
+        self._test_get_redirect(mozwebqa,
+                                '/mobile/aurora/notes/',
+                                '/firefox/android/aurora/notes/',
+                                False)
+        self._test_get_redirect(mozwebqa,
+                                '/mobile/37.0/releasenotes/',
+                                '/firefox/android/37.0/notes/',
```
Comment (kyoshino) [inline tests/test_redirect.py]: Ugh, `global.conf` is right but this test is just wrong. It should be `/firefox/android/37.0/releasenotes/`, of course. Will fix it shortly.

**Comment by kyoshino** [comment]
Fixed the wrong redirect path.

**Comment by jgmize** [comment]
Thanks @kyoshino, I'll test this again as soon as I can borrow a demo environment again.

**Comment by kyoshino** [comment]
Thanks @jgmize!

**Comment by jgmize** [comment]
We finally got a green run! \o/

https://ci.masterfirefoxos.com/view/www.mozilla.org/job/www-demo4.allizom.org-saucelabs/8/console

**Comment by jgmize** [comment]
I've merged https://github.com/mozilla/bedrock/pull/2906, and this is an r+ but I'm going to wait to merge this until right before I push the changes to stage and prod. Thanks again @kyoshino!

**Pull request merged**